### DISCUSSION TITLE
Loading place holder for images

### DISCUSSION THREAD

**Discussion by aminparsa18** [CommunityToolkit/Maui#848] [New Feature Discussions]
As LoadingPlaceHolder and ErrorPlaceHolder was regularly used in project [FFImageLoading](https://github.com/luberda-molinet/FFImageLoading) which is not being maintained anymore I think its nice to have this placeholder image before source is downloaded from server or when there is an issue downloading image so we can show an alternative image instead.

ExtendedImage
              Aspect="AspectFit"
              LoadingPlaceholder="default.png"
              ErrorPlaceholder="error.png"
              Source="example.com/desired.png" />

**Comment by pictos** [comment]
This is planned to be in .NET 8, for Maui. You can engage the progress here https://github.com/dotnet/maui/issues/480

**Reply by EdwardMcFlurry** [reply]
This is frustrating to have it planned for .Net 8

Is there something else we can use in the meantime as a workaround maybe?

**Comment by CliffAgius** [comment]
Closed and answered
